include/swift/Syntax/Serialization/SyntaxDeserialization.h

```diff
@@ -162,9 +162,9 @@ template <> struct MappingTraits<swift::RC<swift::RawSyntax>> {
       StringRef nodeIdString;
       in.mapRequired("id", nodeIdString);
       unsigned nodeId = std::atoi(nodeIdString.data());
-      value =
-          swift::RawSyntax::make(tokenKind, text, leadingTrivia, trailingTrivia,
-                                 presence, /*Arena=*/nullptr, nodeId);
+      value = swift::RawSyntax::make(
+          tokenKind, swift::OwnedString::makeRefCounted(text), leadingTrivia,
+          trailingTrivia, presence, /*Arena=*/nullptr, nodeId);
```

Comment:
How about make `RawSyntax::make()` to receive `StringRef`, then construct make `OwnedString` in it?

Reply:
Ah, you want to control Owned/Unowned in call site. OK then.

Reply:
Yeah, that was my idea behind it.